dependencies.gradle

```diff
@@ -68,6 +68,6 @@ ext {
     revNats = '2.16.14'
     revStan = '2.2.3'
     revFlyway = '10.15.2'
-    revApacheLucene = '10.2.1'
+    revApacheLucene = (System.getenv('SEARCH_ENGINE') == 'OS') ? '9.12.0' : '8.11.1'
```

Comment:
we should add separate dependency and module ofr open search instead of this.

Reply:
Offline convo: provide support for ES or OS not both due to conflicting lucene versions.